hoot-services/pom.xml

```diff
@@ -136,7 +136,7 @@
                     parallelism, but haven't been able to get them to work...so forking instead. -->
                     <!-- parallel>methods</parallel>
                     <useUnlimitedThreads>true</useUnlimitedThreads -->
-                    <forkCount>4</forkCount>
+                    <forkCount>16</forkCount>
```

Comment:
So @dmylov moved dynamic settings of pom.xml.in to db.properties so that hoot-services could be built without the parent hoot dependencies, but I guess we missed that this setting forkCount also got populated by the build.  That may be able to be dynamic, I'll leave that to Dmitriy to investigate.  Were there any other build time parameters replaced in the pom that you recall?